src/util.js

```diff
@@ -187,8 +187,11 @@ util.getFileReplacement = function( src, settings, callback )
     }
     else
     {
-        var result = ( new datauri( util.getInlineFilePath( src, settings.relativeTo ) ) ).content;
-        callback( result === undefined ? new Error( "Local file not found" ) : null, result );
+        var fileName = util.getInlineFilePath( src, settings.relativeTo );
+        var mimetype = mime.getType( fileName );
+        var base64 = fs.readFileSync( fileName, 'base64' );
```

Comment:
can you use the async readFile here? That feels more consistent with the existing callback-ing API

Reply:
Sync version was already used in this module but ok